js/src/UXCapture.js

```diff
@@ -56,8 +63,34 @@ export default class UXCapture {
     }
   }
 
+  /**
+   * Assigns user-specified `onMark` and `onMeasure` callbacks
+   * to UXCapture instance.
+   *
+   * @param {object} configuration
+   */
   config(configuration) {
-    const { onMark = NOOP, onMeasure = NOOP } = configuration;
-    Object.assign(this, { onMark, onMeasure });
+    const { onMark = NOOP, onMeasure = NOOP, interactive = false } = configuration;
+
+    if (interactive) {
+      this.startMark = INTERACTION_START_MARK;
+    }
+
+    Object.assign(this, { onMark, onMeasure, startMark });
+  }
+
+  /**
+   * Updates the `transition-start` mark. `transition-start` is
```

Comment:
I don't think there's anything called `'transition-start'` currently? Should this be `INTERACTION_START_MARK`?